doc/spire_server.md

```diff
@@ -189,7 +189,7 @@ This optional section contains the configurables used by SPIRE Server to expose
 
 | Configuration   | Description                                                                                                               | Default                                          |
 | --------------- | ------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------ |
-| directory_url   | Directory endpoint URL                                                                                                    | "https://acme-v02.api.letsencrypt.org/directory" |
```

Comment:
This text currently gets auto-hyperlinked strangely when run through netlify - which leads to a broken link. Removing the quotes fixes this, but we could also just backtick format (`https://acme-v02.api.letsencrypt.org/directory`) it if preferred, given it feels like this is meant to be a string literal, not a hyperlink.

![image](https://user-images.githubusercontent.com/1670176/163575747-f70e4b1a-0d45-4801-a43f-405cd28103bf.png)